lookout/style/format/tests/test_analyzer.py

```diff
@@ -19,6 +20,35 @@
 Change = NamedTuple("Change", [("base", File), ("head", File)])
 
 
+json_train_config = {
```

Comment:
If we wrap this dict into a function we'll avoid any chance of editing it in some place and creating weird bugs (even though right now everything is deepcopy-ed).

Reply:
+1